packages/process/extension.package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@theia/process",
+  "version": "0.1.1",
+  "description": "Theia process support.",
+  "dependencies": {
+    "@theia/core": "0.1.1",
+    "node-pty": "^0.6.10"
```

Comment:
we can remove node-pty from terminal package, to manage it only here

Reply:
Right I meant to do that and forgot... fixing

Reply:
Fixed that, I was also missing a file, fixed too.